README.md

```diff
@@ -1,4 +1,4 @@
-# Outbound OSS - Guide
```

Comment:
Do you want to add Sofware in here as well?+

Reply:
That's already changed to "Outbound Open Source - Guide"

Reply:
And no, I think that "software" is not required here.